index.html

```diff
@@ -16,8 +16,32 @@
 </head>
 <body>
   <header>
+    <h2 class="container">Code Journal</h2>
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <form>
+        <h2>New Entry</h2><br />
+        <div class="row">
+          <div class="column-half">
+          <img src="images/placeholder-image-square.jpg">
+          </div>
+        <div class="column-half">
+          <label for="title">Title:</label><br />
```

Comment:
Your `for` attribute values don't match the `id` for their form control. You can always test this by clicking on  a label. If it is set up correctly, it should `focus` the input it is tied to.